main.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'app'
+
+def show_menu
+  puts ''
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
```

Comment:
- [ ] After fixing the issue, we have got one another. I cannot creat a person, reason is your inputs are not accepting by create_person method correctly. Kindly fix it. Due to this reason, I was not able to test rental, how correctly created is. Please fix these errors and make sure all the options are working as previous reviewer asked.